packages/ui-components/src/components/multi-select-dropdown/multi-select-dropdown.tsx

```diff
@@ -0,0 +1,141 @@
+	styleUrl: 'multi-select-dropdown.scss',
+	shadow: true
+})
+export class KvMultiSelectDropdown implements IMultiSelectDropdown, IMultiSelectDropdownEvents {
+	/** (required) The text to display as the dropdown placeholder */
+	@Prop({ reflect: true }) placeholder?: string;
+	/** (optional) If `true` the dropdown is opened */
+	@Prop({ reflect: true }) isOpen?: boolean;
+	/** (optional) If `true` the dropdown is loading */
+	@Prop({ reflect: true }) loading?: boolean = false;
+	/** (optional) The icon to display on the dropdown */
+	@Prop({ reflect: true }) icon?: EIconName | EOtherIconName;
+	/** (optional) If `true` the dropdown is searchable */
+	@Prop({ reflect: true }) searchable?: boolean;
+	/** (optional) If `true` dropdown items can be cleared */
+	@Prop({ reflect: true }) selectionClearable?: boolean;
+	/** (optional) If `true` dropdown requires a value to be selected */
+	@Prop({ reflect: true }) required?: boolean;
+	/** (optional) The text to display on the dropdown label */
+	@Prop({ reflect: true }) label?: string;
+	/** (optional) The text to display on the dropdown  */
+	@Prop({ reflect: true }) value?: string;
+	/** (required) The error state for the dropdown */
+	@Prop({ reflect: true }) errorState?: EValidationState;
+	/** (optional) The text to display as help text  */
+	@Prop({ reflect: true }) helpText?: string;
+	/** (optional) If `true` the dropdown is disabled */
+	@Prop({ reflect: true }) disabled?: boolean;
+	/** (required) The text to display when there are no options */
+	@Prop({ reflect: true }) noDataAvailableLabel?: string = MULTI_SELECT_DROPDOWN_NO_DATA_AVAILABLE;
+	/** (optional) The object with the dropdown options */
+	@Prop({ reflect: true }) options?: { [key: string]: IMultiSelectDropdownOption };
+	/** (optional) The array of selected options */
+	@Prop({ reflect: true }) selectedOptions?: string[] = [];
+
+	/** Emitted when the selected options change */
+	@Event() optionsSelected: EventEmitter<string[]>;
+	/** Emitted when the search term changes */
+	@Event() searchChange: EventEmitter<string>;
+
+	private selectOption = (event: CustomEvent<string>) => {
+		const option = event.detail;
+		const currentOptionIndex = this.selectedOptions.findIndex(selectedOption => selectedOption === option);
+
+		if (currentOptionIndex === -1) {
+			this.selectedOptions.push(option);
+		} else {
+			this.selectedOptions.splice(currentOptionIndex, 1);
+		}
+		this.calculateLabelValue();
+
+		this.optionsSelected.emit(this.selectedOptions);
+	};
+
+	private calculateLabelValue() {
+		if (isEmpty(this.options)) {
+			this.value = undefined;
+			return;
+		}
+
+		this.value = this.selectedOptions.reduce((acc, option, currentIndex) => {
+			if (isNil(this.options[option])) {
+				if (currentIndex === this.selectedOptions.length - 1) {
+					acc = acc.slice(0, acc.length - 2);
+				}
+				return acc;
+			}
+			return `${acc + this.options[option].label + (currentIndex !== this.selectedOptions.length - 1 ? ', ' : '')}`;
```

Comment:
We want to put all the options titles on the value? Even if its' unnecessary (if we do ellipsis?)

Reply:
This is just a default behavior, we can pass the title as a prop if we want to do something else.

Reply:
In the use-case I'm using this component on we need the value to be something like `2 filtered`. 

How can we achieve that?